src/context/IntlContext.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
🔧 Can you add a `useIntlContext` function like this one...
https://github.com/aaron-org/snapit-hacktoberfest/blob/8c0b3c3dc2c31899b1005946cdcf96cf09838762/src/context/AuthContext.jsx#L7-L15

Reply:
We decided not to do this per our discussion about the useLocale hook.